Source/Android/src/main/java/com/applovin/godot/AppLovinMAXGodotManager.java

```diff
@@ -0,0 +1,1982 @@
+        disabledAdaptiveBannerAdUnitIds = new HashSet<>( 2 );
+        disabledAutoRefreshAdViewAdUnitIds = new HashSet<>( 2 );
+
+        runOnUiThread( true, new Runnable()
+        {
+            @Override
+            public void run()
+            {
+                safeAreaBackground = new View( getCurrentActivity() );
+                safeAreaBackground.setVisibility( View.GONE );
+                safeAreaBackground.setBackgroundColor( Color.TRANSPARENT );
+                safeAreaBackground.setClickable( false );
+                FrameLayout layout = new FrameLayout( getCurrentActivity() );
+                layout.addView( safeAreaBackground, new FrameLayout.LayoutParams( 0, 0 ) );
+                getCurrentActivity().addContentView( layout, new LinearLayout.LayoutParams( LinearLayout.LayoutParams.MATCH_PARENT, LinearLayout.LayoutParams.MATCH_PARENT ) );
+            }
+        } );
+
+        // Enable orientation change listener, so that the ad view positions can be updated when the device is rotated.
+        getCurrentActivity().getWindow().getDecorView().getRootView().addOnLayoutChangeListener( new View.OnLayoutChangeListener()
+        {
+            @Override
+            public void onLayoutChange(final View view, final int left, final int top, final int right, final int bottom, final int oldLeft, final int oldTop, final int oldRight, final int oldBottom)
+            {
+                boolean viewBoundsChanged = left != oldLeft || right != oldRight || bottom != oldBottom || top != oldTop;
+                if ( !viewBoundsChanged ) return;
+
+                for ( final Map.Entry<String, MaxAdFormat> adUnitFormats : adViewAdFormats.entrySet() )
+                {
+                    positionAdView( adUnitFormats.getKey(), adUnitFormats.getValue() );
+                }
+            }
+        } );
+    }
+
+    public static void setMainActivity(final Activity mainActivity)
+    {
+        if ( mainActivity != null )
+        {
+            AppLovinMAXGodotManager.currentActivity = new WeakReference<>( mainActivity );
+        }
+    }
+
+    public static void setGodotActivity(final Activity godotActivity)
+    {
+        if ( godotActivity != null )
+        {
+            AppLovinMAXGodotManager.godotActivity = new WeakReference<>( godotActivity );
+        }
+    }
+
+    public AppLovinSdk initializeSdkWithCompletionHandler(final String sdkKey,
+                                                          final AppLovinSdkSettings settings,
+                                                          final Listener listener)
+    {
+        final Activity currentActivity = getCurrentActivity();
+        if ( StringUtils.isValidString( sdkKey ) )
+        {
+            sdk = AppLovinSdk.getInstance( sdkKey, settings, currentActivity );
+        }
+        else
+        {
+            sdk = AppLovinSdk.getInstance( settings, currentActivity );
+        }
+
+        sdk.setPluginVersion( "AppLovinMAX-Godot-" + VERSION );
```

Comment:
Convention with other plugins after Unity:

```suggestion
        sdk.setPluginVersion( "Godot-" + VERSION );
```